pyperformance/data-files/benchmarks/bm_yaml/run_benchmark.py

```diff
@@ -0,0 +1,91 @@
+"""
+Script for testing the performance of YAML parsing, using yaml.
+
+This will dump/load several real world-representative objects a few thousand
+times. The methodology below was chosen for was chosen to be similar to
+real-world scenarios which operate on single objects at a time.
+
+This explicitly tests the pure Python implementation in pyyaml, not its C
+extension.
```

Comment:
This is a helpful comment.  It may be worth moving it next to the line where we take this action.  I'm guessing that's the `Loader=yaml.Loader` bit on line 70.